SNOW-502072: Bumped up PythonConnector PATCH version from 2.7.0 to 2.… (#974)
snowflakedb/snowflake-connector-python: DESCRIPTION.rst

-------------------------------------------------------------------------------

- v2.7.1(November 19,2021)

   - Fixed a bug where uploading a streaming file with multiple parts wasn't working 
   - JWT tokens will now be regenerated when requests are retired 
   - Updated url escaping for when uploading to AWS S3 to match how boto3 escapes URLs 
   - Removed a now unused parameter s3_connection_pool_size 
   - Blocked queries will now be considered to be still running 
   - Snowflake specific exceptions will now set Exception args 
   - Fixed a bug where use_s3_regional_url was not set correctly by the connector internally

- v2.7.0(October 25,2021)

>    - Removing cloud sdks.snowflake-connector-python will not install them anymore. Recreate your virtualenv to get rid of unnecessary dependencies.

Reviewer: ```suggestion
   - The cloud sdks.snowflake-connector-python is no longer installed or supported. Recreate your virtual environment to remove any unnecessary dependencies.
```
